mobile/src/helpers/image/pick-image/pick-image.helper.ts

```diff
@@ -0,0 +1,25 @@
+import { Asset, launchImageLibrary } from 'react-native-image-picker';
+
+const pickImage = async (): Promise<Asset | undefined> => {
+  const res = await launchImageLibrary({
+    mediaType: 'photo',
+    selectionLimit: 1,
+  });
+
+  if (!res.assets?.length) {
+    return;
+  }
+
+  return res.assets[0];
```

Comment:
If we will need to select more then one item
```suggestion
  return res.assets;
```

Reply:
i create new helper pickImages for several images

Reply:
we have two helper one for single image second for several images

Reply:
Can't we use only one?

Reply:
or what is the difference between them?